libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/voice/AndroidSpeechPlayer.java

```diff
@@ -33,26 +36,39 @@
     textToSpeech = new TextToSpeech(context, new TextToSpeech.OnInitListener() {
       @Override
       public void onInit(int status) {
-        if (status != TextToSpeech.ERROR) {
-          textToSpeech.setLanguage(new Locale(language));
+        boolean ableToInitialize = status != TextToSpeech.ERROR && language != null;
```

Comment:
For readability, what about negating the condition and calling the boolean `notAbleToInitialize`?

Reply:
I think that makes it more confusing